servicex/transformer/tests/test_arrow_writer.py

```diff
@@ -41,6 +43,24 @@
     from mock import call  # noqa: E402
 
 
+class CaptureHandler(logging.StreamHandler):
+    """
+    Handler that captures messages being logged so that they can be checked
+    """
+    def __init__(self):
+        super().__init__()
+        self.__messages = []
+
+    def emit(self, record: logging.LogRecord) -> None:
+        self.__messages.append(self.format(record))
+
+    def get_messages(self) -> typing.List[str]:
+        return self.__messages
+
+    def reset_messages(self) -> None:
+        self.__messages = []
+
+
```

Comment:
Seems like this `CaptureHandler` class is not used anywhere... does it offer any functionality that is not provided by pytest's `caplog` fixture?